.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: For each commit and PR
+on:
+  push:
+  pull_request:
+
+jobs:
+  validation:
+    runs-on: ubuntu-latest
+    env:
+      CGO_ENABLED: 0
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.6'
+    - name: go fmt
+      run: go get golang.org/x/tools/cmd/goimports; goimports -d . | (! grep .)
+    - name: go vet
+      run: go vet ./...
+    - name: golangci-lint
+      uses: golangci/golangci-lint-action@v1
+      with:
+        version: v1.30
+        args: --skip-dirs cmd/packetflow
+    - name: go test
+      run: go test -v ./...    
+    - name: go test coverage
+      run: go test -coverprofile=coverage.txt ./...
+    - name: upload codecov
+      run: bash <(curl -s https://codecov.io/bash)
+    - name: Build binaries
+      run: go build
+    - name: Upload hegel binary
+      uses: actions/upload-artifact@v2
+      with:
+        name: hegel
+        path: ./hegel
+  docker-images:
+    env:
+      CGO_ENABLED: 0
```

Comment:
this should be done in the validation part right? This isn't really doing anything here.

Reply:
Yes, you are right. This will be removed.